sphinx/json_ref.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+
+import importlib
+import json
+
+import jinja2
+
+from bokeh.plot_object import PlotObject, Viewable
+from bokeh.protocol import serialize_json
+
+ITEM = jinja2.Template("""
+.. code-block:: javascript
+
+    {{ model_json|indent(4) }}
+""")
+
+
+module_names = [
+    'bokeh.models.axes',
+    'bokeh.models.formatters',
+    'bokeh.models.glyphs',
+    'bokeh.models.grids',
+    'bokeh.models.map_plots',
+    'bokeh.models.mappers',
+    'bokeh.models.markers',
+    'bokeh.models.plots',
+    'bokeh.models.ranges',
+    'bokeh.models.renderers',
+    'bokeh.models.sources',
+    'bokeh.models.tickers',
+    'bokeh.models.tools',
+    'bokeh.models.widget',
+]
```

Comment:
Can we get rid of this ^^? I'm sure no one will remember to update this. You can use the following code:
```
name = "bokeh.models"
ns = vars(importlib.import_module(name))
modules = [ obj for obj in ns.values() if isinstance(obj, types.ModuleType) and obj.__name__.startswith(name) ]
sorted(modules, key=lambda obj: obj.__name__)
```
This includes `bokeh.models.widgets.*`, but that can be easily filtered out.